src/PactNet.Native/PactVerifier.cs

```diff
@@ -12,7 +12,7 @@ public class PactVerifier : IPactVerifier
     {
         private readonly IVerifierProvider verifier;
         private readonly PactVerifierConfig config;
-        private readonly IList<string> verifierArgs = new List<string>();
+        protected internal readonly IList<string> verifierArgs = new List<string>();
```

Comment:
I think it's better to add the necessary extra method(s) onto the verifier interface instead of using inheritance to customise the implementation.